app/views/devise/mailer/unlock_instructions.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Hello <%= @resource.email %>!</p>
+
+<p>Your account has been locked due to an excessive number of unsuccessful sign in attempts.</p>
+
```

Comment:
Did you come up with this, or was this a Devise thing? I like it!

Reply:
Devise did this. Devise does a lot for you (which is all fully customizable), but I think it can be dangerous too because it also means there is a lot of code you didn't explicitly write yourself when you need to customize things in a certain way. There is a balance here for sure when considering the pros and cons with a tight deadline.